student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Student < Person
+  # contructor method
+  def initialize(classroom)
+    super(27, 'Harry Winks')
+    @classroom = classroom
+  end
```

Comment:
- [ ] The constructor extends the parent's constructor by adding @classroom and a parameter for it. Also, when creating the super method, you are supposed to extend the functionalities from the Parent class. Kindly consider doing it like this:

```
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```